doc/spec/api.md

```diff
@@ -233,6 +235,9 @@ tag       | The tag for this version of the image.         |
 fsLayers  | A list of layer descriptors (including tarsum) |
 signature | A JWS used to verify the manifest content      |
 
+**TODO**: this suggests to the reader that important API info (the manifest format)
+is not in this spec, but burried in #8093.
```

Comment:
In any case, it should be a full link (https://...), otherwise it will only be clickable when viewed on GitHub